index.html

```diff
@@ -23,6 +23,97 @@ <h2>Software Developer</h2>
+        <h3>Multi-Post Stories</h3>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        <ul class="stack-container">
+            <li class="border-right padding-right">css</li>
+            <li class="border-right padding-right">html</li>          
+            <li class="border-right padding-right">bootstrap</li>         
+            <li class="padding-right">Ruby</li>
+        </ul>
+        <button type="button" class="project-button">See Project</button>
+    </section> 
+    <section class="experience-container">        
+        <img src="./images/Icons/bg.jpg" alt="project image">
+        <h3>Multi-Post Stories</h3>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        <ul class="stack-container">
+            <li class="border-right padding-right">css</li>
+            <li class="border-right padding-right">html</li>          
+            <li class="border-right padding-right">bootstrap</li>         
+            <li class="padding-right">Ruby</li>
+        </ul>
+        <button type="button" class="project-button">See Project</button>
+    </section> 
+    <section class="experience-container">        
+        <img src="./images/Icons/bg.jpg" alt="project image">
+        <h3>Multi-Post Stories</h3>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        <ul class="stack-container">
+            <li class="border-right padding-right">css</li>
+            <li class="border-right padding-right">html</li>          
+            <li class="border-right padding-right">bootstrap</li>         
+            <li class="padding-right">Ruby</li>
+        </ul>
+        <button type="button" class="project-button">See Project</button>
+    </section> 
+    <section class="experience-container">        
+        <img src="./images/Icons/bg.jpg" alt="project image">
+        <h3>Multi-Post Stories</h3>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        <ul class="stack-container">
+            <li class="border-right padding-right">css</li>
+            <li class="border-right padding-right">html</li>          
+            <li class="border-right padding-right">bootstrap</li>         
+            <li class="padding-right">Ruby</li>
+        </ul>
+        <button type="button" class="project-button">See Project</button>
+    </section> 
```

Comment:
- [ ] The whole works section should be in a container (`section`) which should have a grid display.
- [ ] I think each project card should be aligned with flex as stated in the requirements.
- [ ] The See Project buttons in this section should follow the button styling: Active, Hover and Pressed (see design)
- [ ] Kindly make sure this section has the proper background image and font as defined in the 
template